stock_picking_reverse_auto_check_refund/models/stock_return_picking.py

```diff
@@ -0,0 +1,11 @@
+from odoo import models, fields, api
+
+class StockReturnPicking(models.TransientModel):
+    _inherit = "stock.return.picking"
+
+    @api.model
+    def default_get(self, default_fields):
+        res = super(StockReturnPicking, self).default_get(default_fields)
+        for i, k, vals in res.get('product_return_moves', []):
+            vals.update({'to_refund': True})
+        return res
```

Comment:
Shouldn't it suffice if we just assign the default value to this field https://github.com/odoo/odoo/blob/12.0/addons/stock_account/models/stock.py#L775?

Reply:
We can also assign the default value to that field. But I think we assign default value by inheriting function is better way than overriding field in this case(just my POV).